en/08_Changelogs/6.0.0.md

```diff
@@ -41,6 +41,7 @@ This release includes a number of bug fixes to improve a broad range of areas. C
 ### General changes {#api-general}
 
 - [`DataObject::write()`](api:SilverStripe\ORM\DataObject::write()) has a new boolean `$skipValidation` parameter. This can be useful for scenarios where you want to automatically create a new record with no data initially without restricting how developers can set up their validation rules.
+- [`FieldList`](api:SilverStripe\Forms\FieldList) is now strongly typed.
```

Comment:
Probably worth making a small section about this and highlighting things that will require specific attention, e.g. explicitly passing arrays to `addFieldsToTab()`

Reply:
Done